### PR TITLE
T5-423: Add new form

### PR DESCRIPTION
**Pull request by naeluh**

### CODE REVIEW

File: src/components/Form/Form.js [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+      headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+      body: encode({ 'form-name': formName, ...formState }),
+    });
+
+    if (!formResponse.ok) {
+      const message = `An error has occured: ${formResponse.status}`;
+      throw new Error(message);
+    }
+
+    if (formResponse.ok) {
```
Comment (brs14ku) [inline src/components/Form/Form.js:75]: If we're setting `hasSubmitted` to show as an error to limit multiple submissions, why even reset the form? Should we just hide on success?

File: static/functions/submission-created.js [hunk 2]
```diff
@@ -19,6 +19,117 @@ exports.handler = async (event, _context, callback) => {
     return;
   }
 
+  /** Contact Form */
+  if (form_name === 'contact-update') {
+    // handle form contact
+    const { PIPEDRIVE_USER_ID, PIPEDRIVE_KEY } = process.env;
+
+    // pipedrive custom fields
+    const fields = {
```
Comment (brs14ku) [inline static/functions/submission-created.js:28]: Can we make these `env` items instead?

**Review by brs14ku — CHANGES_REQUESTED** [review]
Left you a few comments/q's!

**Comment by naeluh** [comment]
Hey @brs14ku thanks ! I actually updated this over in another PR and I am going to close this one since this has to go into develop. https://github.com/thirdandgrove/thirdandgrove-com-gatsby/pull/624 I updated the code in the ways that you suggested let me know if that works Thanks !

**Comment by brs14ku** [comment]
> Hey @brs14ku thanks ! I actually updated this over in another PR and I am going to close this one since this has to go into develop. #624 I updated the code in the ways that you suggested let me know if that works Thanks !

Approved the other. Thanks!

**Pull request closed without merging**